coalib/results/SourceRange.py

```diff
@@ -10,7 +10,10 @@ class SourceRange(TextRange):
     @enforce_signature
     def __init__(self,
                  start: SourcePosition,
-                 end: (SourcePosition, None)=None):
+                 end: (SourcePosition, None)=None,
+                 text=None,
+                 start_position=None,
```

Comment:
@AbdealiJK  AFAIK file is picked up from the SourcePosition, we don't need to pass that in SourceRange?